### PR TITLE
MD simulation & analysis workflows

### PR DESCRIPTION
**Pull request by chrisbarnettster**
@simonbray 's gromacs workflow
and @tsenapathi 's MD analysis with bio3d  workflow

### CODE REVIEW

**Comment by chrisbarnettster** [comment]
forgot the yml...

File: training/computational-chemistry/bio3danalysis/MD_Analysis_using_Bio3D.ga [hunk 1]
```diff
@@ -0,0 +1 @@
...rmsd\": \"{\\\"__current_case__\\\": 0, \\\"sele\\\": \\\"calpha\\\"}\", \"__rerun_remap_job_id__\": null, \"pdbin\": \"{\\\"__class__\\\": \\\"RuntimeValue\\\"}\", \"chromInfo\": \"\\\"/opt/galaxy/tool-data/shared/ucsc/chrom/?.len\\\"\", \"dcdin\": \"{\\\"__class__\\\": \\\"RuntimeValue\\\"}\"}", "id": 4, "tool_shed_repository": {"owner": "chemteam", "changeset_revision": "75fd897bd85d", "name": "bio3d_rmsd", "tool_shed": "toolshed.g2.bx.psu.edu"}, "uuid": "8088ee87-34b5-4499-9419-4fcceb2f2a4a", "errors": null, "name": "RMSD Analysis", "post_job_actions": {"HideDatasetActionrmsd_hist_plot": {"output_name": "rmsd_hist_plot", "action_type": "HideDatasetAction", "action_arguments": {}}, "HideDatasetActionrmsd_plot": {"output_name": "rmsd_plot", "action_type": "HideDatasetAction", "action_arguments": {}}}, "label": null, "inputs": [{"name": "pdbin", "description": "runtime parameter for tool RMSD Analysis"}, {"name": "dcdin", "description": "runtime parameter for tool RMSD Analysis"}], "position": {"top": 403, "left": 735}, "annotation": "", "content_id": "toolshed.g2.bx.psu.edu/repos/chemteam/bio3d_rmsd/bio3d_rmsd/2.3", "type": "tool"}}, "annotation": "", "a_galaxy_workflow": "true"}
```
Comment (bgruening) [inline training/computational-chemistry/bio3danalysis/MD_Analysis_using_Bio3D.ga]: Mh, the other workflow was more readable :(

Comment (chrisbarnettster) [reply training/computational-chemistry/bio3danalysis/MD_Analysis_using_Bio3D.ga]: will pretty format with jq and I am still fixing that yml, running in planemo now

Comment (erasche) [reply training/computational-chemistry/bio3danalysis/MD_Analysis_using_Bio3D.ga]: No problem @chrisbarnettster, just ping us when it's ready!

**Comment by chrisbarnettster** [comment]
done 👍  @erasche @bgruening

**Comment by bgruening** [comment]
Cool works for me! Thanks a bunch @chrisbarnettster!

@simonbray you could yours as well if you like.

**Comment by bgruening** [comment]
I will add an entry to the front-page.

**Pull request merged**